core/src/com/unciv/logic/battle/Battle.kt

```diff
@@ -254,9 +254,11 @@ class Battle(val gameInfo:GameInfo) {
     }
 
     private fun captureCivilianUnit(attacker: ICombatant, defender: ICombatant){
-        // barbarians don't capture civilians, City-states don't capture settlers
-        if(attacker.getCivInfo().isBarbarian()
-                || (attacker.getCivInfo().isCityState() && defender.getName()==Constants.settler)){
+        // barbarians, City-states, OCC don't capture civilians don't capture settlers
+        var cannotCaptureSettler = (attacker.getCivInfo().isBarbarian()
+                || attacker.getCivInfo().isCityState()
+                || attacker.getCivInfo().isPlayerOneCityChallenger())
+        if(cannotCaptureSettler && defender.getName()==Constants.settler){
```

Comment:
This means that barbarians will be able to capture other civilians, not what we want...